presto-router/etc/router-config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "user-credentials": "",
```

Comment:
Shouldn't we leave this parameter out entirely if the standard "out of the box" configuration has no authentication configured?